functions/Set-DbaStartupParameter.ps1

```diff
@@ -164,7 +164,7 @@ After the work has been completed, we can push the original startup parameters b
 	process {
 		
 		#Get Current parameters:
-		$currentstartup = Get-DbaStartupParameter -SqlInstance $sqlinstance -Credential $Credential
+		$currentstartup = Get-DbaStartupParameter -SqlServer $SqlServer -Credential $Credential
```

Comment:
This also switches back to SqlServer instead of SqlInstance even though instance is supported in Get-DbaStartupParameter